<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-2CJ4JG5" failures="1" tests="1" name="Default test(failed)(failed)" time="0.089" errors="0" timestamp="2022-11-19T08:27:17 GMT-08:00">
  <testcase classname="testNGfailedtestcaseexcuetion.FailTC1" name="w3" time="0.02">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testNGfailedtestcaseexcuetion.FailTC1.w3(FailTC1.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- w3 -->
  <testcase classname="testNGfailedtestcaseexcuetion.FailTC1" name="w2" time="0.0">
    <ignored/>
  </testcase> <!-- w2 -->
  <testcase classname="testNGfailedtestcaseexcuetion.FailTC1" name="w1" time="0.0">
    <ignored/>
  </testcase> <!-- w1 -->
</testsuite> <!-- Default test(failed)(failed) -->
